quantuminspire/sdk/circuit.py

```diff
@@ -0,0 +1,109 @@
+"""Module containing the Quantum Circuit class."""
+
+from pathlib import Path
+from types import TracebackType
+from typing import Optional, Type
+
+import openql
+from openql import Kernel, Platform, Program
+
+
+class Circuit:
+    """A container object, interacting with OpenQL and storing cQASM
+    internally.
+
+    A circuit wraps OpenQL to handle the boilerplate code for platform,
+    program and kernels. These objects can still be used.
+    """
+
+    def __init__(self, platform: str, program: str) -> None:
```

Comment:
I recommend using platform_name and program_name consistently.

This is especially important for the platform. The OpenQL interface for Platform is a bit complicated, identifying platform names, platform files, Python objects containing a platform, string containing a (JSON-representation of a) platform.